src/abstractdataframe/io.jl

```diff
@@ -34,13 +35,17 @@ function printtable(io::IO,
     end
     for i in 1:n
         for j in 1:p
-            if ! (etypes[j] <: Real)
-                print(io, quotemark)
-                escapedprint(io, df[i, j], "\"'")
-                print(io, quotemark)
-            else
-                print(io, df[i, j])
-            end
+	    if ! (isna(df[i,j]))
```

Comment:
`isna(df[j], i)` to avoid introducing type instability. (It's a subtle one, and one of the reasons people are looking forward to using `NullableArray`s with more obvious behavior.)